qemu_v8.mk

```diff
@@ -189,6 +256,35 @@ soc-term:
 soc-term-clean:
 	$(MAKE) -C $(SOC_TERM_PATH) clean
 
+################################################################################
+# mkimage - create images to be loaded by U-Boot
+################################################################################
+# Without the objcopy, the uImage will be 10x bigger.
+$(KERNEL_UIMAGE): linux | $(BINARIES_PATH)
+	${AARCH64_CROSS_COMPILE}objcopy -O binary \
+					-R .note \
+					-R .comment \
+					-S $(LINUX_PATH)/vmlinux \
+					$(BINARIES_PATH)/linux.bin
+	$(MKIMAGE_PATH)/mkimage -A arm64 \
+				-O linux \
+				-T kernel \
+				-C none \
+				-a $(KERNEL_LOADADDR) \
+				-e $(KERNEL_ENTRY) \
+				-n "Linux kernel" \
+				-d $(BINARIES_PATH)/linux.bin $(KERNEL_UIMAGE)
+
+$(ROOTFS_UGZ): buildroot | $(BINARIES_PATH)
```

Comment:
`$(ROOTFS_UGZ): u-boot buildroot | $(BINARIES_PATH)`

(`mkimage` is generated by target `u-boot`)

Reply:
Thanks @jforissier , I will add the dependency